src/open_inwoner/core/views.py

```diff
@@ -1,4 +1,8 @@
+from urllib.parse import urlencode
+
 from django.shortcuts import render
+from django.urls import reverse
```

Comment:
@pi-sigma Perhaps you could look at this part again to see if I'm doing this correctly - I'm doing a bunch of reverseURL things here in order to try and get the Begeleiders URL.
It seems to work...